sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/ThriftServerQueryTestSuite.scala

```diff
@@ -103,7 +103,8 @@ class ThriftServerQueryTestSuite extends SQLQueryTestSuite with SharedThriftServ
     // SPARK-42921
     "timestampNTZ/datetime-special-ansi.sql",
     // SPARK-47264
-    "collations.sql"
+    "collations.sql",
+    "pipe-operators.sql"
```

Comment:
why it doesn't work in thrift-server?

Reply:
Update on this, the test flaked out again:

<img width="1122" alt="image" src="https://github.com/user-attachments/assets/ed1f3925-4ca3-4f08-a79f-e065e7750818">

It seems this test suite is not sorting the output result rows in the absence of an ORDER BY clause, like the other SQL test suite variants. I'm adding it back to the blocklist for now.